### PR TITLE
style: builds hackathon badge

### PR DESCRIPTION
**Pull request by Prerna290**
# PR Template

## Type

- ( ) Fix
- ( ) Refactor
- ( ) Style
- ( ) Docs
- (X) Feat
- ( ) Hotfix

## Screenshots
image
image
image

### CODE REVIEW

File: apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/community-build-card/community-build-card.component.html [hunk 1]
```diff
@@ -16,10 +16,10 @@
         <app-badge
           *ngIf="communityBuild.hackathon_team"
           [text]="'Hackathon'"
-          [fontColor]="'com-text-yellow-700'"
-          [fontSize]="'small'"
+          [fontColor]="'com-text-tChocolate-Traditional'"
+          [fontSize]="'mini'"
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/community-build-card/community-build-card.component.html]: let's make mini to extra-small or xs

File: apps/shared-components/badge/badge.component.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
   class="{{ position }} {{ color }}"
 >
   <div>
-    <p [ngClass]="{ 'small-text': fontSize == 'small' }" class="{{ fontColor }}">
+    <p [ngClass]="{ 'small-text': fontSize == 'small', 'mini-text': fontSize == 'mini' }" class="{{ fontColor }}">
```
Comment (ArshdeepGrover) [inline apps/shared-components/badge/badge.component.html]: let's make mini to extra-small or xs

File: apps/shared-components/badge/badge.component.scss [hunk 3]
```diff
@@ -16,6 +16,14 @@ div {
...
     }
+
+    &.mini-text {
+      @apply com-py-2px com-px-1 com-leading-3 com-text-8px com-font-medium;
+
+      nb-icon {
+        @apply com-text-sm;
+      }
+    }
   }
```
Comment (ArshdeepGrover) [inline apps/shared-components/badge/badge.component.scss]: let's make mini to extra-small or xs

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

File: apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/community-build-card/community-build-card.component.html (hunk 1 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/community-build-card/community-build-card.component.html]: extra small is already made with 10px font size, its 8px font size

File: apps/shared-components/badge/badge.component.html (hunk 2 above)
Comment (Prerna290) [reply apps/shared-components/badge/badge.component.html]: xtra small is already made with 10px font size, its 8px font size

File: apps/shared-components/badge/badge.component.scss (hunk 3 above)
Comment (Prerna290) [reply apps/shared-components/badge/badge.component.scss]: extra small is already made with 10px font size, its 8px font size

File: apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/community-build-card/community-build-card.component.html (hunk 1 above)
Comment (arpansac) [reply apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/community-build-card/community-build-card.component.html]: xxs should be the double extra small

**Review by ArshdeepGrover — APPROVED** [review]

**Pull request merged**